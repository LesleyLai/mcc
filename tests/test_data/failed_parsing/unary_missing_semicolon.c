int main(void)
{
    return !10
}