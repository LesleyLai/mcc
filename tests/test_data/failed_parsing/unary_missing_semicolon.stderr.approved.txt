{{filename}}:4:1: Error: Expect ;
4 | }
  | ^

